xray/src/generation.rs

```diff
@@ -203,6 +258,38 @@ impl ColoringStrategy for PointColorColoringStrategy {
     }
 }
 
+struct HeightStddevColoringStrategy {
+    per_column_data: FnvHashMap<(u32, u32), OnlineStats>,
+    max_stddev: f32,
+}
+
+impl HeightStddevColoringStrategy {
+    fn new(max_stddev: f32) -> Self {
+        HeightStddevColoringStrategy {
+            max_stddev, 
+            per_column_data: FnvHashMap::default(),
+        }
+    }
+}
+
+impl ColoringStrategy for HeightStddevColoringStrategy {
+    fn process_discretized_point(&mut self, p: &Point, x: u32, y: u32, _: u32) {
```

Comment:
Except XRayColoringStrategy, none of the strategy impl are using `z` argument.
How about args: ..., or tuples or Options<>?

Reply:
I am not sure what you are asking for here.

Reply:
Am suggesting either we use variadic functions using a tuple
https://github.com/rust-lang/rfcs/issues/376

Or use Options, whichever is more Rustful.

Reply:
- variadic functions are not implemented in Rust so far. It seems it is also still quite controversial.
- Option seems wrong, because it is always passed by the caller, but not always used by the callee. Option is used if a value is sometimes passed, sometimes not.
- We cannot get rid of `z` because it is used by XRayColoringStrategy